docs/root/configuration/http_filters/tap_filter.rst

```diff
@@ -0,0 +1,134 @@
+.. _config_http_filters_tap:
+
+Tap
```

Comment:
Can we document here how streaming matching works? E.g. if you have a [request header, response header] match condition, and you've seen and matched only request headers, do you dump, then if you see a non-response header match you don't dump that?

Reply:
Streaming tapping is not implemented yet for output. It is implemented in the sense that if you match on request headers, but there is no response, it will still dump the matching request headers at the end. I will clarify.